tools/downloader/list_topologies.yml

```diff
@@ -3233,3 +3233,125 @@ topologies:
     output: "Security/object_detection/barrier/0106/dldt/"
     framework: dldt
     license: https://raw.githubusercontent.com/opencv/open_model_zoo/master/LICENSE
+
+  - name: "mobilenet-v2-pytorch"
+    description: >-
+      MobileNet V2 is image classification model pretrained on ImageNet dataset. This
+      is a PyTorch implementation of MobileNetV2 architecture as described in
+      the paper "Inverted Residuals and Linear Bottlenecks: Mobile Networks for Classification,
+      Detection and Segmentation" <https://arxiv.org/pdf/1801.04381.pdf>.
+
+      The model input is a blob that consists of a single image of "1x3x224x224"
+      in RGB order. 
+
+      The model output is typical object classifier for the 1000 different classifications
+      matching with those in the ImageNet database.
+    task_type: "classification"
+    files:
+      - name: MobileNetV2.py
+        sha256: 7071e01e07e5f1b6127a2628d1be9c65cc20edfa745f4f4eb6354e7b27f5b444
+        source: https://raw.githubusercontent.com/tonylins/pytorch-mobilenet-v2/fa86a85e4026caa217fe8d62f370848195cedf4b/MobileNetV2.py
+      - name: mobilenet-v2.pth.tar
+        sha256: ecbe2b568c8602549fa9e1d5833c63848f490a48d92e5d224d1eb2063e152cf8
+        size: 14205652
+        source:
+          $type: google_drive
+          id: 1jlto6HRVD3ipNkAl1lNhDbkBp7HylaqR
+    output: "classification/mobilenet/v2/pytorch/"
+    framework: pytorch
+    pytorch_to_onnx:
+      - --model-name=mobilenet-v2
+      - --model-path=classification/mobilenet/v2/pytorch
+      - --weights=classification/mobilenet/v2/pytorch/mobilenet-v2.pth.tar
+      - --input-shape=1,3,224,224
+      - --output-file=classification/mobilenet/v2/pytorch/mobilenet-v2.onnx
+      - --input-names=data
+      - --output-names=prob
+    model_optimizer_args:
+      - --model_name=mobilenet-v2
+      - --framework=onnx
+      - --data_type=FP32
+      - --input=data
+      - --mean_values=data[123.675,116.28,103.53]
+      - --scale_values=data[58.624,57.12,57.375]
+      - --output=prob
+      - --input_model=$dl_dir/mobilenet-v2.onnx
```

Comment:
It is now possible to specify a separate directory for converter output, so you can't use `dl_dir` here. You'll need to add another variable, e.g. `conv_dir`.

Reply:
All models (in pth and onnx formats) will be in same, `dl_dir`, directory

Reply:
Well, you shouldn't put the ONNX model in `dl_dir` - it's not something that we download.